source/developers/references/aspects_home.rst

```diff
@@ -0,0 +1,13 @@
+################
+Aspects
```

Comment:
```suggestion
Aspects, the Open edX Analytics Solution
```

I'm not sure if this is the _correct_ title, but we need a little bit more than "Aspects". It's new, and people don't necessarily know what it is. Any thoughts?

Reply:
good call, the openedx-aspects repo says `Learner analytics for Open edX`, does that work?

Reply:
Yeah that's perfect

Reply:
Although Open edX should be used as an adjective, _always_, not a noun. Can we change it to "Open edX Learner Analytics" or "Learner Analytics for the Open edX platform"? cc @crathbun428 @bmtcril

Reply:
I've been using "Learner Analytics for the Open edX platform"

Reply:
I like Open edX Learner Analytics